calico.go

```diff
@@ -500,5 +500,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	skel.PluginMain(cmdAdd, cmdDel, cniSpecVersion.All)
+	skel.PluginMain(cmdAdd, nil, cmdDel,
+		cniSpecVersion.PluginSupports("0.1.0", "0.2.0", "0.3.0", "0.3.1"),
```

Comment:
Note: this excludes v0.4.0 so the framework will not allow us to be called for CHECK (until we add that later).